hackathon_site/hardware/test_api.py

```diff
@@ -318,6 +319,114 @@ def test_hardware_quantity_success(self):
         self.assertEqual(expected_unique_hardware_counts, actual_unique_hardware_counts)
 
 
+class IncidentListsViewTestCase(SetupUserMixin, APITestCase):
+    def setUp(self):
+        super().setUp()
+        self.team = Team.objects.create()
+        self.order = Order.objects.create(status="Cart", team=self.team)
+        self.hardware = Hardware.objects.create(
+            name="name",
+            model_number="model",
+            manufacturer="manufacturer",
+            datasheet="/datasheet/location/",
+            notes="notes",
+            quantity_available=4,
+            max_per_team=1,
+            picture="/picture/location",
+        )
+
+        self.other_hardware = Hardware.objects.create(
+            name="other",
+            model_number="otherModel",
+            manufacturer="otherManufacturer",
+            datasheet="/datasheet/location/other",
+            quantity_available=10,
+            max_per_team=10,
+            picture="/picture/location/other",
+        )
+        self.order_item = OrderItem.objects.create(
+            order=self.order, hardware=self.hardware,
+        )
+
+        self.order_item2 = OrderItem.objects.create(
+            order=self.order, hardware=self.other_hardware,
+        )
+
+        self.incident = Incident.objects.create(
+            state="Broken",
+            description="Description",
+            order_item=self.order_item,
+            time_occurred="2022-08-08 00:00:00",
```

Comment:
It looks like Django is throwing a warning because it expects datetimes to be timezone aware. Easy fix:

```python
from datetime import datetime
...
from django.conf import settings
...

self.incident = Incident.objects.create(
    state="Broken",
    description="Description",
    order_item=self.order_item,
    time_occurred=datetime(2022, 8, 8, tzinfo=settings.TZ_INFO),
)
```

Reply:
I have encountered this bug before, and when I set the year from 2021 to 2022, it just disappeared on my end.